pyorbital/tests/test_geoloc.py

```diff
@@ -190,6 +190,16 @@ def test_viirs(self):
         np.testing.assert_allclose(geom.fovs,
                                    expected_fovs, rtol=1e-2, atol=1e-2)
 
+        geom = viirs(1, chn_pixels=3)
+        expected_fovs = np.array([
+            np.tile(np.array([[0.98, -0., -0.98]]), [32, 1]),
+            np.tile(np.array([[0., -0., 0]]), [32, 1])], dtype=np.float64)
+
+        np.testing.assert_allclose(geom.fovs,
+                                   expected_fovs, rtol=1e-2, atol=1e-2)
+        #print(geom.fovs)
+        #print(geom.fovs.shape)
+
```

Comment:
```suggestion
    def test_viirs_defaults(self):
        """Test the definition of the viirs instrument with default slicing."""
        geom = viirs(1, chn_pixels=3)
        expected_fovs = np.array([
            np.tile(np.array([[0.98, -0., -0.98]]), [32, 1]),
            np.tile(np.array([[0., -0., 0]]), [32, 1])], dtype=np.float64)

        np.testing.assert_allclose(geom.fovs,
                                   expected_fovs, rtol=1e-2, atol=1e-2)

```

Reply:
Our new-ish standard practice is to make separate tests actually separate rather than having a ton of independent test in a single test method.

Reply:
OK, suggested changes just pushed in a squashed commit.